Sources/StreamChatUI/Common Views/ChatChannelCreateNewButton.swift

```diff
@@ -11,12 +11,13 @@ public typealias ChatChannelCreateNewButton = _ChatChannelCreateNewButton<NoExtr
 /// A Button subclass that should be used for creating new channels.
 open class _ChatChannelCreateNewButton<ExtraData: ExtraDataTypes>: Button, UIConfigProvider {
     override public func defaultAppearance() {
-        defaultIntrinsicContentSize = .init(width: 44, height: 44)
+        super.defaultAppearance()
         setImage(uiConfig.images.newChat, for: .normal)
     }
-
-    open var defaultIntrinsicContentSize: CGSize?
-    override open var intrinsicContentSize: CGSize {
-        defaultIntrinsicContentSize ?? super.intrinsicContentSize
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        widthAnchor.pin(equalToConstant: 60).isActive = true
+        heightAnchor.pin(equalToConstant: 60).isActive = true
```

Comment:
I don't think we should do this at all. Let the underlying UIButton instance decide its size.